x/twap/api_test.go

```diff
@@ -74,7 +75,9 @@ func (s *TestSuite) TestGetBeginBlockAccumulatorRecord() {
 			actualRecord, err := s.twapkeeper.GetBeginBlockAccumulatorRecord(s.Ctx, tc.poolId, tc.baseDenom, tc.quoteDenom)
 
 			if tc.expError != nil {
-				s.Require().Equal(tc.expError, err)
+				if !strings.Contains(err.Error(), tc.expError.Error()) {
+					s.Require().Equal(tc.expError, err)
+				}
```

Comment:
This basically relaxes the error checking condition to substring matching for now.

Reply:
If the substring is not in the string, why is it necessary to check if they are equal later? Wouldn't that automatically be a false case? For instance this case: `ExpErr: F, Err: ABCD` 

Also you think  we can use `ErrorContainsf`instead? 
`s.Require().ErrorContainsf(err,  tc.expectError, err.Error(), "formatted")`

Reply:
@stackman27 I think he just did a quick fix for CI. 

I implemented ErrorContains instead of ErrorContainsf, but had similar thinking as you, lmk if you agree

https://github.com/osmosis-labs/osmosis/pull/2641/commits/ac1796cbda18f76ddd77ef7c39db1df86ce53bd2

Reply:
looks solid!